php/Terminus/Models/MachineToken.php

```diff
@@ -17,15 +19,17 @@ public function __construct($attributes, array $options = []) {
 
   /**
    * Deletes machine token
-   *
-   * @return array
+   * @return null
```

Comment:
Stack Exchange / PHPDoc wiki page says to omit @return if the return type is void.

http://stackoverflow.com/questions/2061550/phpdoc-return-void-necessary

I have no objection if we knowingly wish to diverge from the standard, though.